5.5.0_2.2/Dockerfile

```diff
@@ -0,0 +1,82 @@
+FROM buildpack-deps:jessie
+
+MAINTAINER Jack <jack@expert360.com>
+
+# ====================
+# Install Nodejs 5.5.0 LTS
```

Comment:
Where'd you find the Dockerfile for 5.5.0? I can't find it in the official repo.